src/policy/compiler.rs

```diff
@@ -1308,14 +1297,16 @@ mod tests {
     #[test]
     fn compile_timelocks() {
         // artificially create a policy that is problematic and try to compile
+        /*
```

Comment:
Creating an issue to fix this.

Reply:
I don't remember commenting out this code. I wonder if some unknown-to-me vim binding did it.

Reply:
Oh, no, I remember now. The point is that this policy used to cause an error when you compiled it, but now you just can't construct it.

I should delete this test entirely.